public/data/avneesh0612.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Avneesh Agarwal",
-  "bio": "I am a web developer, designer, content creator, open source contributor, and student.",
+  "bio": "I am a web developer, designer, content creator, open source contributor, and a student.",
```

Comment:
Not sure if this `a` is needed as there's already an `a` at the beginning. 🤔

Reply:
Avneesh asked be to add it tho 😓